WordPress/build.gradle

```diff
@@ -181,6 +190,23 @@ android {
     buildFeatures {
         viewBinding true
     }
+    sourceSets {
```

Comment:
Warning (⚠️): As discussed, this block was added automatically by the IDE, but it is not required as this is the default behavior anyway. Let's remove it.